File: 프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.py
def solution(dots):
    answer = 0
    
    dots = sorted(dots, key=lambda x:(x[0],x[1]))
    
    y = dots[1][1] - dots[0][1] 
    x = dots[2][0] - dots[0][0]
    
    return x * y

File: 프로그래머스/0/120860. 직사각형 넓이 구하기/직사각형 넓이 구하기.java
import java.util.*;

class Solution {
    public int solution(int[][] dots) {
        int answer = 0;
        
        Arrays.sort(dots, (a,b) -> {
            if (a[0] != b[0]) return a[0] - b[0];
            return a[1] - b[1];
            
        });
        
        int y = dots[1][1] - dots[0][1];
        int x = dots[2][0] - dots[0][0];
        
        return x*y;
    }
}